app/models/sg_sequence_prompts.rb

```diff
@@ -22,12 +22,18 @@ def add_visual_prompts(definition,p_name)
   end
 
   def initial_prompt_from_hash(defs)
-    self.initial_prompt = defs['text']
     if self.respond_to?(:answer_with_label) && defs['label']
       self.answer_with_label = true
     end
-    if self.respond_to?(:title)
-      self.title          = defs['text'] # sequences with titles don't include them in their hash, so we use the initial_prompt
+    prompt = defs['text']
```

Comment:
This was the #2 cause for copy failures.  Even if a sequence had a title, it was overwritten by the magic *_from_hash method of the initial prompt.  The problem here was that often times the initial prompt was longer than would fit in the DB field for title.  (Also, authors lost their titles ….)